test/cypress/integration/pages/your-financial-path-to-graduation.js

```diff
@@ -0,0 +1,75 @@
+import { PfcFinancialPathToGraduation } from '../../pages/paying-for-college/your-financial-path-to-graduation/';
+
+const page = new PfcFinancialPathToGraduation();
+
+describe( 'Paying For College', () => {
+  describe( 'Your Financial Path To Graduation', () => {
+    it( 'should display total costs', () => {
+      page.open();
+      page.click( 'Get started' );
+      page.setText( 'search__school-input', 'Harvard' );
+      page.selectProgram( 'type', 'bachelors' );
```

Comment:
An idea for more assertions, as mentioned by @Scotchester: 
If you choose 'type', 'graduate', there should be options in the `program-select` selection box.

Reply:
I added assertions using `page.getText` to verify total costs.
Also, I added tests for all program types: certificate, associates, graduate, as well as bachelors.